MapboxCoreNavigation/RouteController.swift

```diff
@@ -216,6 +253,27 @@ extension RouteController: CLLocationManagerDelegate {
         lastUserDistanceToStartOfRoute = Double.infinity
     }
     
+    func pushAndResetLocationCounters() {
+        var eventDictionary = MGLMapboxEvents.addDefaultEvents(routeProgress: routeProgress, sessionIdentifier: sessionIdentifier)
+        eventDictionary["startTimestamp"] = startTimestamp
+        eventDictionary["feedbackType"] = "reroute"
+        eventDictionary["locationsBefore"] = convertLocastionsObject(locations: Array(userLocationForRerouteEvent.prefix(rerouteIndexLocationBefore)))
+        eventDictionary["locationsAfter"] = convertLocastionsObject(locations: Array(userLocationForRerouteEvent.suffix(from: rerouteIndexLocationBefore)))
+        eventDictionary["userLocatioBeforeAfterReroute"] = userLocationForRerouteEvent
```

Comment:
This property isn't in the spec so the event may be discarded by the API. What's the thought behind including this?